service/src/main/kotlin/fi/espoo/evaka/application/ApplicationForm.kt

```diff
@@ -255,8 +255,8 @@ data class ApplicationForm(
                         postOffice = guardian.postOffice ?: ""
                     ),
                     futureAddress = null,
-                    phoneNumber = "",
-                    email = ""
+                    phoneNumber = guardian.phone ?: "",
```

Comment:
Is it ok to default to a known phone number here?

Reply:
yeah I think that makes sense